west.yml

```diff
@@ -243,7 +243,7 @@ manifest:
       groups:
         - hal
     - name: hal_ti
-      revision: 2e7b95ad079e9f636884eedc6853e6ad98b85f65
+      revision: pull/51/head
```

Comment:
This PR includes dts files, why are they in this repo and not added to zephyr?

Reply:
I'm seeing having full SOC level pinctrl files are done by SOC. It seems like Nordic has very simple files that are done for just a few peripherals, and there's not an easy way to show all configurations, such as ADC pins, etc. How is this handled?

Reply:
ADC is handled by pinctrl